### PR TITLE
limesurvey: 2.05_plus_141210 -> 3.17.1+190408, init module

### PR DESCRIPTION
**Pull request by aanderse**
###### Motivation for this change
Ultimately to remove usage of mod_php in all nixos modules.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by aanderse** [comment]
@Izorkin Sorry it took a while, but I finally got around to it. This is what I had in mind.
@GrahamcOfBorg test limesurvey

**Comment by aanderse** [comment]
@GrahamcOfBorg test limesurvey

File: nixos/modules/services/web-apps/limesurvey.nix [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+    };
+
+in
+{
+  # interface
+
+  options.services.limesurvey = {
+    enable = mkEnableOption "Limesurvey web application.";
+
+    httpd = mkOption {
```
Comment (aanderse) [inline nixos/modules/services/web-apps/limesurvey.nix]: Possibly more fitting to call this to virtualHost instead of httpd?

**Comment by ivan** [comment]
@matthewbauer this broke the build of the manual:

```
building '/nix/store/i6skz5jjrlf7miwbj5d7f51cq5k282rb-nixos-manual-combined.drv'...
building '/nix/store/g3412qglf20d840y8z8as6g98r2c5k6g-ibus-mozc-2.23.2815.102.drv'...
release-notes/rl-1909.xml:122: element section: validity error : ID sec-release-19.09-incompatibilities already defined
         xml:id="sec-release-19.09-incompatibilities">
                                                     ^

manual-combined.xml:68172: element section: validity error : ID sec-release-19.09-incompatibilities already defined
 68168     </listitem>
 68169    </itemizedlist>
 68170   </section>
 68171
 68172   <section xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xi="http://www.w3.org/2001/XInclude" version="5.0" xml:id="sec-release-19.09-incompatibilities">
 68173    <title>Backward Incompatibilities</title>
 68174

w3.org/2001/XInclude" version="5.0" xml:id="sec-release-19.09-incompatibilities"

                                                                               ^
unpacking sources
unpacking source archive /nix/store/wbzb243c93kqxmb61ydi604343l4m79g-source
source root is source
patching sources
applying patch /nix/store/zrr7zi41kln766y4gvh4ib9rfn11h7ys-fix-paths.patch
patching file Makefile.am
patching file data/80-udisks2.rules
patching file modules/zram/udiskslinuxmanagerzram.c
patching file src/tests/install-udisks/runtest.sh
patching file src/tests/integration-test
Hunk #1 succeeded at 418 (offset 4 lines).
Hunk #2 succeeded at 1094 (offset 15 lines).
Hunk #3 succeeded at 1107 (offset 15 lines).
patching file src/tests/storadectl/runtest.sh
patching file src/tests/test.c
Hunk #4 succeeded at 144 with fuzz 1 (offset -1 lines).
Hunk #5 succeeded at 197 (offset -2 lines).
applying patch /nix/store/1635v514qqr6kcsj4c4rql9k6bqqm8f9-force-path.patch
patching file src/main.c
Hunk #1 succeeded at 142 with fuzz 1 (offset 2 lines).
configuring
autoreconf: Entering directory `.'
autoreconf: running: autopoint --force

manual-combined.xml:68121: element filename: Relax-NG validity error : Did not expect element filename there
 68117     <listitem>
 68118      <para>
 68119       PostgreSQL now uses
 68120       <filename class="directory">/run/postgresql</filename> as its socket
 68121       directory instead of <filename class="directory">/tmp</filename>. So
 68122       if you run an application like eg. Nextcloud, where you need to use
 68123       the Unix socket path as the database host name, you need to change it

manual-combined.xml:68073: element section: Relax-NG validity error : Did not expect element section there
 68069    This section lists the release notes for each stable version of NixOS and
 68070    current unstable revision.
 68071   </para>
 68072   <section xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xi="http://www.w3.org/2001/XInclude" version="5.0" xml:id="sec-release-19.09">
 68073   <title>Release 19.09 (&#x201C;Loris&#x201D;, 2019/09/??)</title>
 68074
 68075   <section xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xi="http://www.w3.org/2001/XInclude" version="5.0" xml:id="sec-release-19.09-highlights">

manual-combined.xml:68073: element section: Relax-NG validity error : Element appendix has extra content: section
 68069    This section lists the release notes for each stable version of NixOS and
 68070    current unstable revision.
 68071   </para>
 68072   <section xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xi="http://www.w3.org/2001/XInclude" version="5.0" xml:id="sec-release-19.09">
 68073   <title>Release 19.09 (&#x201C;Loris&#x201D;, 2019/09/??)</title>
 68074
 68075   <section xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xi="http://www.w3.org/2001/XInclude" version="5.0" xml:id="sec-release-19.09-highlights">

manual-combined.xml:3: element info: Relax-NG validity error : Element book has extra content: info
     1  <?xml version="1.0"?>
     2  <book xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xi="http://www.w3.org/2001/XInclude" version="5.0" xml:id="book-nixos-manual">
     3   <info>
     4    <title>NixOS Manual</title>
     5    <subtitle>Version 19.09
     6    </subtitle>
     7   </info>

manual-combined.xml fails to validate
builder for '/nix/store/i6skz5jjrlf7miwbj5d7f51cq5k282rb-nixos-manual-combined.drv' failed with exit code 3
cannot build derivation '/nix/store/a2fwcf48yq57qdy1pf37pnz2bzarrfcm-nixos-manpages.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/3wxm8kn3jzski7hiawzjlbksbwffk5b2-nixos-manual-html.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/5nm723vhzjvg0fclk8j5f0fxj6ipxm12-system-path.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/px8l0wpdl23y9l2h3xpfppaxmkz7jpx3-nixos-system-ra-19.09.git.a5bf968.drv': 1 dependencies couldn't be built
error: build of '/nix/store/px8l0wpdl23y9l2h3xpfppaxmkz7jpx3-nixos-system-ra-19.09.git.a5bf968.drv' failed
```

**Comment by aanderse** [comment]
@ivan Thanks for catching. Looks like a merge conflict gone wrong. Fixing now.

**Pull request merged**